dwitter/static/main.css

```diff
@@ -475,6 +475,7 @@ iframe {
   transition: opacity .1s ease-in-out, margin .1s step-end;
   opacity: 0;
   height: 0;
+  margin-top: -9999px;  # remove the hover-trigger while invisible
```

Comment:
Actually, this isn't valid css comment syntax, but right here it seems like browsers just ignore it anyway, so it doesn't break anything.

`/* remove the hover-trugger while invisible */` would be correct.

Reply:
Fixed